<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CTimeCardDialog</class>
 <widget class="QDialog" name="CTimeCardDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>451</width>
    <height>139</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>计时卡</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>卡号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCardNumber"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>卡自编号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCardSelfNo"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>登记者：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtRecorder">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>411</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>卡类型：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxCardType">
      <item>
       <property name="text">
        <string>EMID卡</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MIFARE卡</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>收费标准：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxStandard"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>卡状态：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbxCardStatus">
      <item>
       <property name="text">
        <string>启用</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>挂失</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>保存</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>退出</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
